src/libraries/System.Runtime.InteropServices/tests/ComDisabled/AssemblyInfo.cs

```diff
@@ -0,0 +1,6 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+[assembly: ActiveIssue("https://github.com/dotnet/runtime/issues/34748", TestPlatforms.Windows, TargetFrameworkMonikers.Netcoreapp, TestRuntimes.Mono)]
```

Comment:
Does this issue also apply to these new tests? It looks like the issue was from Mono not having built-in COM support on Windows. That seems odd to apply for tests that are disabling built-in COM support - if the issue does also apply, it should be updated to call out what exactly needs to be addressed for these tests.

Reply:
Also, I would recommend not using an assembly attribute like this to disable all tests in an assembly. That results in the tests still being built, packaged, and sent to a helix machine which will go through test discovery for each test case to determine that it should not be run and then upload test artifacts / results (of not running anything).

There is the `IgnoreForCI` property (see eng/testing/tests.targets) to build the test, but not package / send to helix or the `ProjectExclusions` item (see src/libraries/tests.proj) to skip it altogether.